app/api/v1/mediator/ballot.py

```diff
@@ -164,55 +191,54 @@ def validate_ballot(
 
 
 def _get_election_parameters(
-    election_id: Optional[str], request: BaseBallotRequest
+    election_id: Optional[str],
+    request_data: BaseBallotRequest,
```

Comment:
request_data:  BaseBallotRequest = Body(...),

We probably need to set this as the body since in the function you are using it as a body.